src/container/containerManager.ts

```diff
@@ -50,10 +51,13 @@ export class ContainerManager {
     }
```

Comment:
deployment.template.debug.json is not listed when triggering buildSolution command via command palette.

Reply:
Neither is the generateConfig command.